examples/src/main/java/com/mapbox/navigation/examples/core/ReplayHistoryActivity.kt

```diff
@@ -107,8 +109,11 @@ class ReplayHistoryActivity : AppCompatActivity() {
                 style,
                 mapboxNavigation,
                 navigationMapboxMap,
-                mapboxReplay
+                mapboxReplay,
+                HistoryRecorder(mapboxNavigation)
             )
+            navigationContext.historyRecorder.startRecording()
```

Comment:
Are we recording history files from history files replayed? 🤯 

Jokes aside, I think it'd be great to have an option in the app's settings so the examples that integrate the `HistoryRecorder` will enable / disable it.

Also, `HistoryRecorder` stores history files in `com.mapbox.navigation.examples > cache > history-cache` it'd be great that those show up as well in the `Select history` list.

Reply:
> Are we recording history files from history files replayed? 🤯
> Jokes aside, I think it'd be great to have an option in the app's settings so the examples that integrate the HistoryRecorder will enable / disable it.

haha yeah only adding it here for an example, but you're right that is not practical. Good suggestion to add it to the settings, I'll look into it. Also good suggestion to make it show up in select history 👍

Reply:
Hey look it's already there! 🙃  🤔 

<img width="382" alt="Screen Shot 2020-08-31 at 10 38 57 AM" src="https://user-images.githubusercontent.com/3021882/91749368-3836b880-eb76-11ea-90ba-8b9cdac63447.png">